commands/service_delete.go

```diff
@@ -37,7 +37,7 @@ mesg-core service delete --all`,
 
 func (c *serviceDeleteCmd) preRunE(cmd *cobra.Command, args []string) error {
 	if len(args) == 0 && !c.all {
-		return errors.New("at least one service id must be provided (or run with --all flag)")
+		return errors.New("at least one service id must be provided (or run with --all flag).")
```

Comment:
Don't use dot at the end of errors. In case of error delegation/concentration someone may put `:` or `-`. It's up to printing method/func to put dot or not